packages/stencil-library/src/components/alert/alert.e2e.ts

```diff
@@ -1,10 +1,119 @@
-import { newE2EPage } from '@stencil/core/testing';
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+const renderAndGetAlertDiv = async (content: string): Promise<{ alertDiv: E2EElement; page: E2EPage }> => {
```

Comment:
[DUST] This helper function is great for pulling out the render code and grabbing the component to test!